src/scss/components/optional/_user-message.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * User Message Component
+ * =================================
+ * Used to display the user message banner on the page.
+ *
+ * *The `c-userMessage` component is an optional mixin within Fozzie — if you'd like to use it in your project you can include it by adding `@include userMessage();` into your SCSS dependencies file.*
+ */
+
+@mixin userMessage() {
+    .c-userMessage {
+        color: #fff;
+        background-color: $orange;
+        max-width: 100%;
+        margin-top: spacing(x2);
+    }
+
+    .c-userMessageContainer {
+        max-width: 350px;
+        margin: 0 auto;
+        padding: spacing(x2) 0;
+        display: grid;
+        grid-template-columns: 40px auto;
+
+        @include media('>=narrow') {
+            max-width: 540px;
+        }
+
+        @include media('>=mid') {
+            max-width: 650px;
+            margin: 0 auto;
+            padding: spacing(x2) 0 spacing(x2) 36px;
+        }
+
+        svg {
+            fill: white;
+            width: 28px;
+            height: 28px;
+            display: block;
+            align-self: center;
+        }
+    }
+
+    .c-userMessageText {
+        @include font-size(base);
```

Comment:
Does this need to be set because the text isn't inside a `p` tag? (just interested as can't see the markup here 😄 )

Reply:
It looks like it doesn't need to be set, because it is in a `p` tag 👍